src/plugins/event-plugins/__integ__/DomEventPlugin.test.ts

```diff
@@ -193,9 +190,9 @@ test('when two elements identified by a CSS selector are clicked then CSS select
             JSON.parse(e.details).cssLocator === '[label="label1"]'
     );
 
-    for (let i = 0; i < events.length; i++) {
-        let eventType = events[i].type;
-        let eventDetails = JSON.parse(events[i].details);
+    for (const event of events) {
```

Comment:
This not not so safe to do with arrays

Consider using Array.forEach

Reply:
Hey, could you explain why this isn't a safe operation with arrays? This was recommended by TSLint and I'm not able to find much information on why `forEach` is preferred - although according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of, it has compatibility issues with IE, so I will go ahead and make the change.

Reply:
I forget the reasons 100%, but I believe in some cases (and some browsers) `for..of` returns the `length` value as well.